crypto/evp/pmeth_lib.c

```diff
@@ -250,6 +250,13 @@ EVP_PKEY_CTX *EVP_PKEY_CTX_new_id(int id, ENGINE *e)
     return int_ctx_new(NULL, e, id);
 }
 
+#ifndef OPENSSL_NO_CNSM
+EVP_PKEY_CTX *EVP_PKEY_CTX_new_pkey_id(EVP_PKEY *pkey, int id, ENGINE *e)
```

Comment:
Is it necessary to export the `int_ctx_new` function?
That's effectively what is being done here.

If it **must** be exported, renaming the original function is a better way.
I'm suspicious about it needing to be though.